README.md

```diff
@@ -53,6 +53,30 @@ See [lock-filename](#lock-filename) for more information on changing the default
 lockfile paths and [pip-compile-constraint](#pip-compile-constraint) for more
 information on syncing dependency versions across environments.
 
+### Updating Dependencies
+
+Passing arguments like `--upgrade` / `--upgrade-package` to `pip-compile` is
```

Comment:
For anyone who doesn't read the whole sentence, the entire takeaway of this is that you can pass an `--upgrade` flag. Which you can't. I suggest not mentioning it at all, or mentioning at the end in parentheses.